packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -365,19 +376,36 @@ contract VotingV2 is
         // the associated slashing rates as a function of the total staked, total votes and total correct votes. Note
         // that this method in almost all cases will only need to traverse one request as slashing trackers are updated
         // on every commit and so it is not too computationally inefficient.
-        uint256 priceRequestIdsLength = priceRequestIds.length;
-        for (uint256 i = lastRequestIndexConsidered; i < priceRequestIdsLength; i = unsafe_inc(i)) {
-            if (deletedRequestJumpMapping[i] != 0) i = deletedRequestJumpMapping[i] + 1;
-            Request memory request = priceRequestIds[i];
-            PriceRequest storage priceRequest = priceRequests[request.requestId];
+        for (uint256 requestIndex = lastRequestIndexConsidered; requestIndex < priceRequestIds.length; requestIndex++) {
```

Comment:
the changes in this loop are very subtle. it'll be useful to look at the unit tests to understand how these factors in rolls.